src/kw_remote.sh

```diff
@@ -11,6 +11,16 @@ function remote_main()
     exit 0
   fi
 
+  if [[ "$1" =~ -l|--list ]]; then
```

Comment:
The correct way to handle parameters is inside the parser function `parse_config_options`. Here, you should check for `options_values` variable.